dashel/dashel-posix.cpp

```diff
@@ -1078,6 +1078,13 @@ namespace Dashel
 				ioctl(fd, TIOCMBIS, &iFlags);
 			else
 				ioctl(fd, TIOCMBIC, &iFlags);
+			
+            //Ignore SIGHUP from this point ownward.
+            //TODO: we should probably reset the signal to its initial state at some point
```

Comment:
Rather than saying "at some point", suggest something more precise such as "for example when no more serial ports are in use".